### PR TITLE
DOC/STY: docstrings for instrument modules (3 of 3)

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Updates docstring style and import order in preparation for flake8-docstrings and hacking tests.

Adds `flake8-docstrings` and `hacking` to CI test environment.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Local run of flake8.

## Test Configuration
* Operating system: Mac OS X 10.15.7
* Version number: Python 3.8.2
* `flake8-docstrings` and `hacking`

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes 
- [na] Update zenodo.json file for new code contributors

### CODE REVIEW

File: pysatNASA/instruments/cnofs_plp.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
-"""Supports the Planar Langmuir Probe (PLP) onboard the Communication
+"""Module for the C/NOFS PLP innstrument.
```
Comment (jklenzing) [inline pysatNASA/instruments/cnofs_plp.py:2]:
```suggestion
"""Module for the C/NOFS PLP instrument.
```

File: pysatNASA/instruments/icon_euv.py [hunk 2]
```diff
@@ -156,7 +158,7 @@ def clean(self):
 
 
 def load(fnames, tag=None, inst_id=None, keep_original_names=False):
-    """Loads ICON EUV data using pysat into pandas.
+    """Load ICON EUV data using pysat into xarray.
```
Comment (aburrell) [inline pysatNASA/instruments/icon_euv.py:161]:
```suggestion
    """Load ICON EUV data into xarray.Dataset object and pysat.Meta objects.
```

File: pysatNASA/instruments/icon_ivm.py [hunk 3]
```diff
@@ -216,7 +217,7 @@ def clean(self):
 
 
 def load(fnames, tag=None, inst_id=None, keep_original_names=False):
-    """Loads ICON IVM data using pysat into pandas.
+    """Load ICON IVM data using pysat into pandas.
```
Comment (aburrell) [inline pysatNASA/instruments/icon_ivm.py:220]:
```suggestion
    """Load ICON IVM data into pandas.DataFrame and pysat.Meta objects.
```

File: pysatNASA/instruments/icon_mighti.py [hunk 4]
```diff
@@ -273,7 +275,7 @@ def _clean_vars(var_list, flag, min_level):
 
 
 def load(fnames, tag=None, inst_id=None, keep_original_names=False):
-    """Loads ICON FUV data using pysat into pandas.
+    """Load ICON FUV data using pysat into xarray.
```
Comment (aburrell) [inline pysatNASA/instruments/icon_mighti.py:278]:
```suggestion
    """Load ICON FUV data into xarray.Dataset and pysat.Meta objects.
```

File: pysatNASA/instruments/ses14_gold.py [hunk 5]
```diff
@@ -136,7 +138,7 @@ def clean(self):
 
 
 def load(fnames, tag=None, inst_id=None):
-    """Loads GOLD NMAX data using pysat into xarray
+    """Load GOLD NMAX data using pysat into xarray.
```
Comment (aburrell) [inline pysatNASA/instruments/ses14_gold.py:141]:
```suggestion
    """Load GOLD NMAX data into xarray.Dataset and pysat.Meta objects.
```

**Review by rstoneback — APPROVED** [review]
Looks good to me. Thanks @jklenzing. Not trying to get ahead of any other potential comments from @aburrell.

**Pull request merged**